spec/features/courses/accredited_body/new_spec.rb

```diff
@@ -0,0 +1,107 @@
+require "rails_helper"
+
+feature "Edit accredited body", type: :feature do
+  let(:current_recruitment_cycle) { build(:recruitment_cycle) }
+  let(:provider) { build(:provider) }
+  let(:course) { build(:course, provider: provider) }
+  let(:new_accredited_body_page) { PageObjects::Page::Organisations::Courses::NewAccreditedBodyPage.new }
+  let(:new_accredited_body_search_page) { PageObjects::Page::Organisations::Courses::NewAccreditedBodySearchPage.new }
+  let(:accrediting_provider_1) { build(:provider) }
+  let(:accrediting_provider_2) { build(:provider) }
+  let(:accredited_bodies) {
+    [
+      { "provider_name": accrediting_provider_1.provider_name, "provider_code" => accrediting_provider_1.provider_code },
+      { "provider_name": accrediting_provider_2.provider_name, "provider_code" => accrediting_provider_2.provider_code },
+    ]
+  }
+
+  before do
+    stub_omniauth
+    stub_api_v2_resource(provider)
+    stub_api_v2_build_course
+    stub_api_v2_build_course(level: "primary")
+
+    2.times do
```

Comment:
As mentioned in #708, can we `visit signin_path` instead of, essentialy, port-knocking here? :) (see https://github.com/DFE-Digital/manage-courses-frontend/pull/708#issuecomment-547426801)